.github/workflows/python-app.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: [ "master" ]
+    paths: ['backend/**', '.github/**']
+  pull_request:
+    branches: [ "master" ]
+    paths: ['backend/**', '.github/**']
+
+permissions:
+  contents: read
+
+env:
+  DAY_OF_WEEK: Monday
+  DB_SERVER: localhost
+  DB_PORT: 3306
+  DB_USER: test
+  DB_PASSWORD: test
+  DATABASE: test
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: ./backend
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.10"
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 pytest
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Lint with flake8
+      run: |
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Test with pytest
+      run: |
+        pytest
```

Comment:
Will require changes to this line in the future to prepare the test database (changes are introduced in https://github.com/is212g7t5/spm/pull/9)

Reply:
Hmm ok can you create a ticket and place in the backlog? I'm not too sure what to change it to.

Reply:
Created at https://is212g7t5.atlassian.net/browse/G7T5-46